portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,21 +29,25 @@
 import java.util.ArrayList;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String text = request.getParameter("comment-input");
-    String timeStamp = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(new Date());
-    String user = "Anonymous";
+    final String COMMENT_INPUT_PARAMETER_NAME = request.getParameter("comment-input");
```

Comment:
Oh, the comment input shouldn't be a constant. I was referring to the String "comment-input" so you'd have some class-wide constant `static final String COMMENT_INPUT_PARAMETER_NAME = "comment-input";` which you'd then use here like `String comment = request.getParameter(COMMENT_INPUT_PARAMETER_NAME);` Does that make sense?